### ISSUE TITLE
StatusBar in Screenshots

### ISSUE THREAD

**Issue by PowerPaloma** [kif-framework/KIF#1149]
There's a way to also have statusBar in screenshots ?

**Comment by kenji21** [comment]
Yes, see https://github.com/openium/SwiftiumTestingKit/blob/master/STKTestAppTests/STKSoloTests.swift#L192

we use it with https://github.com/TitouanVanBelle/XCTestHTMLReport in our CI to get a folder with all screenshots (also available in Test report of Xcode on your local machine)

**Comment by justinseanmartin** [comment]
It looks like you're using `XCUIScreen.main.screenshot().image`, so I guess that method is able to enumerate the status bar and render it to the image. We should potentially swap out our rendering code in `-[UIApplication(KIFAdditions) writeScreenshotForLine:inFile:description:error:]` to use that to get the image instead.

I unfortunately don't have a lot of free time to update the library right now, but I'd be happy to help get a change merged if you're willing to contribute.
